asreview/base_model.py

```diff
@@ -29,12 +37,21 @@ def default_param(self):
         dict:
             Dictionary with parameter: default_value
         """
-        signature = inspect.signature(self.__init__)
-        return {
-            k: v.default
-            for k, v in signature.parameters.items()
-            if v.default is not inspect.Parameter.empty
-        }
+        cur_class = self.__class__
+        default_parameters = sig_to_param(inspect.signature(self.__init__))
+        while cur_class != BaseModel:
```

Comment:
I don't have any clue what's going on here.

Is `isinstance` better?

Reply:
It is going through all the classes in the inheritance tree to find default parameters. So, isinstance would return True, but class == BaseModel only returns true if we arrive at the root of the tree.